### ISSUE TITLE
PostgreSQL driver Psycopg2 issues

### ISSUE THREAD

**Issue by sidprobstein** [sidprobstein/swirl-search#55]
The psycopg2 driver for PostgreSQL has been removed from requirements.txt. This is because PostgreSQL must be installed, and the pg_config executable must be in the path.

If that is the case, simply do:

```
pip install psycopg2
python swirl.py restart core
```

Note that there is error handling to take care of the case where the PostgreSQL connector is requested, but the package is not installed. An error will appear in the message section of the search object.

**Comment by devbazregari** [comment]
hi bro I hope you're doing great. would you please tell me where(in which file or line )  the error happen?

**Comment by sidprobstein** [comment]
Hi! Same here.

The error occurs in [swirl/connectors/postgres.py](https://github.com/sidprobstein/swirl-search/blob/main/swirl/connectors/postgresql.py) on line 23:

```
try:
    import psycopg2
except ImportError as e:
    logger.error(f"postgresql.py: Error: can't load psycopg2: {e}, see https://github.com/sidprobstein/swirl-search/wiki/4.-Object-Reference#postgresql")
```

The error is caught, but it still appears in logs etc. 

The reason I removed it from requirements.txt is that if you don't actually have PostgreSQL installed you can't run pg_config which is required for psycopg2 to work... suggestions most appreciated :-)

**Comment by sidprobstein** [comment]
Fixed in 1.7; removed from swir.connectors.__init__ and documentation updated to reflect this
